src/rpc/core_rpc_server.cpp

```diff
@@ -113,6 +113,7 @@ namespace cryptonote
   {
     m_restricted = restricted;
     m_net_server.set_threads_prefix("RPC");
+    m_net_server.set_connection_filter(&m_p2p);
```

Comment:
That makes me wince, but I think I'll just extract that from p2p later unless you want to do it now :)

Reply:
I thought that would require moving too much code around, and this was easy...

Reply:
Yes, I like how this patch's pretty small :) I'll have a go at it later, hopefully it won't be too large either.